pandas/tests/frame/methods/test_fillna.py

```diff
@@ -857,3 +857,100 @@ def test_pad_backfill_deprecated(func):
     df = DataFrame({"a": [1, 2, 3]})
     with tm.assert_produces_warning(FutureWarning):
         getattr(df, func)()
+
+
+@pytest.mark.parametrize(
```

Comment:
im a bit late here, but these tests look really similar to the series ones.  is there a viable way to share them?

Reply:
Yes - but where would they go? Just all in frame/methods?

Reply:
usually we put them in the frame/methods/test_foo.py file and parametrize over frame_or_series